tests/smoke_tests/test_basic.py

```diff
@@ -566,7 +566,7 @@ def test_kubernetes_context_failover():
                 'kubectl get namespaces --context kind-skypilot | grep test-namespace || '
                 '{ echo "Should set the namespace to test-namespace for kind-skypilot. Check the instructions in '
                 'tests/test_smoke.py::test_kubernetes_context_failover." && exit 1; }',
-                'sky show-gpus --cloud kubernetes --region kind-skypilot | grep H100 | grep "1, 2, 3, 4, 5, 6, 7, 8"',
+                'sky show-gpus --cloud kubernetes --region kind-skypilot | grep H100 | grep "1, 2, 4, 8"',
```

Comment:
Why do we make the change here? In k8s, we do all different number of GPUs? @romilbhardwaj

Reply:
This is what I execute:

```bash
sky local up
kubectl label node --overwrite skypilot-control-plane skypilot.co/accelerator=h100 --context kind-skypilot

kubectl proxy &

sleep 5

curl --header "Content-Type: application/json-patch+json" \
  --request PATCH \
  --data '[{"op": "add", "path": "/status/capacity/nvidia.com~1gpu", "value": "8"}]' \
  http://localhost:8001/api/v1/nodes/skypilot-control-plane/status

kubectl create namespace test-namespace --context kind-skypilot
kubectl config set-context kind-skypilot --namespace=test-namespace --context kind-skypilot


kind create cluster --name kind-cluster-2
kubectl config use-context kind-skypilot

sky check
```

But the `sky show-gpus --cloud kubernetes --region kind-skypilot` only shows  `1, 2, 4, 8`